src/mcl_3dl.cpp

```diff
@@ -256,7 +256,7 @@ class MCL3dlNode
     try
     {
       geometry_msgs::TransformStamped trans =
-          tfbuf_.lookupTransform(frame_ids_["odom"], msg->header.frame_id, ros::Time(0));
+          tfbuf_.lookupTransform(frame_ids_["odom"], msg->header.frame_id, ros::Time::now(), ros::Duration(0.1));
```

Comment:
Using `now()` has same problem as using `Time(0)`.
Timestamp of the incoming message must be used here.